### DISCUSSION TITLE
Welcome to Jellyflix Discussions!

### DISCUSSION THREAD

**Discussion by jdk-21** [jellyflix-app/jellyflix#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Apprisco** [comment]
Genuinely amazing how fast Jellyflix is, it loads shows in seconds instead of 30

**Comment by Rohan-K12** [comment]
can to add control to skip to next episode while watching a series

**Reply by jdk-21** [reply]
You mean a button when watching an episode?
When do you think should it appear?

**Comment by Rohan-K12** [comment]
Yes while watching episode and while one episode is going to end a pop up
should come to skip to next episode and we can customise the time when it
should appear like when the episode watching have remaining 1min or
something

**Reply by jdk-21** [reply]
I'm not a fan of giving users settings for such things. It should be intuitive enough that you don't need a specific setting. But I'm ok with having a bottom pop up before the episode ends.

**Reply by sevenrats** [reply]
I get wanting to skip to the next episode, but the proposed solution is unusual. Why not a conventional “next” button like media players have had for 30 years?
